tests/contrib/botocore/test.py

```diff
@@ -94,16 +95,15 @@ def test_s3_put_object(self):
         spans = writer.pop()
         assert spans
         span = spans[0]
-        self.assertEquals(len(spans), 2)
-        self.assertEquals(span.get_tag('aws.operation'), 'CreateBucket')
-        self.assertEquals(span.get_tag(http.STATUS_CODE), '200')
-        self.assertEquals(span.service, "test-botocore-tracing.s3")
-        self.assertEquals(span.resource, "s3.createbucket")
-        self.assertEquals(spans[1].get_tag('aws.operation'), 'PutObject')
-        self.assertEquals(spans[1].resource, "s3.putobject")
+        self.assertEqual(len(spans), 2)
+        self.assertEqual(span.get_tag('aws.operation'), 'CreateBucket')
+        self.assertEqual(span.get_tag(http.STATUS_CODE), '200')
+        self.assertEqual(span.service, "test-botocore-tracing.s3")
+        self.assertEqual(span.resource, "s3.createbucket")
+        self.assertEqual(spans[1].get_tag('aws.operation'), 'PutObject')
+        self.assertEqual(spans[1].resource, "s3.putobject")
         self.assertIsNotNone(spans[1].get_tag('params'))
-        params = json.loads(spans[1].get_tag('params'))
-        self.assertDictEquals(params, dict(Key='foo', Bucket='mybucket'))
+        self.assertEqual(spans[1].get_tag('params'), stringify(dict(Key='foo', Bucket='mybucket')))
```

Comment:
I wonder if we shouldn't have `spans[1].get_tag('params.Key')` so in the UI it gets rendered as:

```
params: 
    Key: 'foo'
    Bucket: 'mybucket'
```

Instead of:

```
params: {'Key'='foo', 'Bucket'='mybucket'}
```